src/context/yaml/handlers/roles.js

```diff
@@ -6,17 +6,9 @@ async function parse(context) {
 }
 
 async function dump(context) {
-  // remove empty descriptions
+  // nothing to do, set default empty
   return {
-    roles: [
-      ...(context.assets.roles || []).map((role) => {
-        if (role.description === null) {
-          delete role.description;
-        }
-
-        return role;
-      })
-    ]
```

Comment:
Also needs to figure out if this is a regression.

Reply:
https://github.com/auth0/auth0-deploy-cli/blob/master/test/context/yaml/roles.test.js#L61-L81